### ISSUE TITLE
netns-bridge@.service

### ISSUE THREAD

**Issue by p00h** [Jamesits/systemd-named-netns#22]
...contains
```
Before=network.target network-online.target
```
but in case of DHCP option enabled the service always fails because of dhclient does not supposed to be operational unless both network targets are finished.

**Comment by Jamesits** [comment]
Yes, I know that Linux doesn't support long interface names. This is now documented.
